okhttp-tests/src/test/java/okhttp3/EventListenerTest.java

```diff
@@ -116,6 +118,59 @@
     assertEquals(expectedEvents, listener.recordedEventTypes());
   }
 
+  @Test public void successfulCallEventSequenceForEnqueue() throws IOException,
+      InterruptedException {
+    server.enqueue(new MockResponse()
+        .setBody("abc"));
+
+    Call call = client.newCall(new Request.Builder()
+        .url(server.url("/"))
+        .build());
+
+    final CountDownLatch l = new CountDownLatch(1);
```

Comment:
Can’t have this. `l` is forbidden as a variable name, especially on a line that also has `1` !